src/xrGame/alife_object.cpp

```diff
@@ -22,72 +22,80 @@ void CSE_ALifeObject::spawn_supplies(LPCSTR ini_string)
-        xr_vector<u32> OnlyOne;
+        xr_vector<u32> spawnLoadouts;
 
         pcstr lname = ai().game_graph().header().level(ai().game_graph().vertex(m_tGraphID)->level_id()).name().c_str();
 
-        for (u32 k = 0; ini.r_line(LOADOUT_SECTION, k, &itmSection, &V); k++)
+        for (u32 k = 0; ini.r_line(loadoutSection, k, &itmSection, &V); k++)
         {
             // If level=<lname> then only spawn items if object on that level
             if (strstr(V, "level=") != nullptr)
             {
                 if (strstr(V, lname) != nullptr)
-                    OnlyOne.push_back(k);
+                    spawnLoadouts.push_back(k);
             }
             else
             {
-                OnlyOne.push_back(k);
+                spawnLoadouts.push_back(k);
             }
         }
 
-        if (!OnlyOne.empty())
+        if (!spawnLoadouts.empty())
         {
-            s32 sel = Random.randI(0, OnlyOne.size());
-            if (ini.r_line(LOADOUT_SECTION, OnlyOne.at(sel), &itmSection, &V))
+            s32 sel = Random.randI(0, spawnLoadouts.size());
+            if (ini.r_line(loadoutSection, spawnLoadouts.at(sel), &itmSection, &V))
             {
                 VERIFY(xr_strlen(itmSection));
                 if (pSettings->section_exist(itmSection))
                 {
-                    u32 spawn_count = 1;
+                    u32 spawnCount = 1;
                     bool bScope = false;
                     bool bSilencer = false;
                     bool bLauncher = false;
-                    float f_cond = 1.0f;
-                    int i_ammo_type = 0, n = 0;
+                    float fCond = 1.0f;
+                    int iAmmoType = 0, n = 0;
 
                     if (V && xr_strlen(V))
                     {
                         n = _GetItemCount(V);
                         if (n > 0)
                         {
                             string64 tmp;
-                            spawn_count = atoi(_GetItem(V, 0, tmp)); //count
+                            spawnCount = atoi(_GetItem(V, 0, tmp)); //count
                         }
 
-                        if (!spawn_count) spawn_count = 1;
-                        if (nullptr != strstr(V, "cond="))
-                            f_cond = static_cast<float>(atof(strstr(V, "cond=") + 5));
-                        bScope = nullptr != strstr(V, "scope");
-                        bSilencer = nullptr != strstr(V, "silencer");
-                        bLauncher = nullptr != strstr(V, "launcher");
-                        if (nullptr != strstr(V, "ammo_type="))
-                            i_ammo_type = atoi(strstr(V, "ammo_type=") + 10);
-                    }
+                        bScope = is_spawn_supplies_flag_set(V, "scope");
+                        bSilencer = is_spawn_supplies_flag_set(V, "silencer");
+                        bLauncher = is_spawn_supplies_flag_set(V, "launcher");
 
+                        if (!spawnCount) spawnCount = 1;
```

Comment:
Code style: always put if body on a new line.
```suggestion
                        if (!spawnCount)
                            spawnCount = 1;
```